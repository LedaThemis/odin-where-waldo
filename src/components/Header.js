import '../styles/Header.css';

const Header = () => {
  return (
    <header id="header">
      <h1 id="header--title">Where's Waldo?</h1>
    </header>
  );
};

export default Header;
