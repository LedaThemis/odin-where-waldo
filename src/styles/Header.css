#header {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  background-color: #394053;
  color: #f7f0f5;

  padding: 1rem 0;
}

#header--title {
  margin: 0;
}

#header--notice {
  margin: 0;
}
