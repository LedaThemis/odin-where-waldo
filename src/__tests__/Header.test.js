import rerenderer from 'react-test-renderer';
import Header from '../components/Header';

test('should render correctly', () => {
  const tree = rerenderer.create(<Header />).toJSON();

  expect(tree).toMatchSnapshot();
});
