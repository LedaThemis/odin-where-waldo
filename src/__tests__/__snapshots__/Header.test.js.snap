// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<header
  id="header"
>
  <h1
    id="header--title"
  >
    Where's Waldo?
  </h1>
  <p
    id="header--notice"
  >
    Click Anywhere!
  </p>
</header>
`;
